core/chains/evm/client/chain_client.go

```diff
@@ -3,18 +3,17 @@ package client
 import (
 	"context"
 	"math/big"
+	"sync"
 	"time"
 
 	"github.com/ethereum/go-ethereum"
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/types"
-	"github.com/ethereum/go-ethereum/rpc"
+	ethrpc "github.com/ethereum/go-ethereum/rpc"
```

Comment:
Why alias this? There are no other conflicts currently, and we're already in the evm subtree :shrug: 
```suggestion
	"github.com/ethereum/go-ethereum/rpc"
```

Reply:
Oh, was it in order to use `rpc` as a variable name? That's not a very strong reason. I think it would be more idiomatic to use just `r` or `s` (for selected) in most cases.

Reply:
Sounds good, will update to use `r`